src/core_plugins/console/public/src/directives/sense_settings.js

```diff
@@ -16,6 +16,16 @@ require('ui/modules')
         $scope.kbnTopNav.close();
       };
 
+      const self = this;
+
+      function onEnter(event) {
+        if (event.which === 13) {
+          self.apply();
+        }
+      }
```

Comment:
I don't know if there has been a style discussion or decision on arrow functions but if you use the arrow style I think you can get rid of the self variable (suggestion only and curious to hear if there is a reason not to use arrow style).